src/OmniSharp.Roslyn.CSharp/Services/Signatures/SignatureHelpService.cs

```diff
@@ -96,27 +97,43 @@ public SignatureHelpService(OmniSharpWorkspace workspace)
                 var invocation = node as InvocationExpressionSyntax;
                 if (invocation != null && invocation.ArgumentList.Span.Contains(position))
                 {
+                    var i = await document.GetSemanticModelAsync();
```

Comment:
probably give `i` a slightly better name.

Reply:
Call this `semanticaModel` and replace the call to `await document.GetSemanticModelAsync()` below with the variable as well. There's no need to call it twice.